examples/cpp/infer_multiple_networks/CMakeLists.txt

```diff
@@ -60,8 +60,8 @@ if (APPLE)
 endif()
 
 target_link_libraries(
-    infer_multiple_networks
-    ngraph_bridge
+    infer_multiple_networks -Wl,--whole-archive
+    ngraph_bridge_static -Wl,--no-whole-archive
```

Comment:
note to self: helps gtest pass by exposing all functions etc

some info here: https://github.com/tensorflow/tensorflow/issues/4242

```
TensorFlow uses a static registration mechanism (in particular behind the REGISTER_OP() and REGISTER_KERNEL_BUILDER() macros) in the implementation of the kernel library, and the default build rules create a single binary that contains all kernels. Without --whole-archive, these static registrations would be stripped, and the kernels would not be registered. 
```